test/unit/kaocha/version_check_test.clj

```diff
@@ -0,0 +1,22 @@
+
+(ns kaocha.version-check-test
+  (:require [clojure.test :refer :all]
+            [kaocha.test-helper]
+            [kaocha.version-check :as v]))
+
+
```

Comment:
Not a blocker, but please be mindful of use of whitespace

https://nextjournal.com/lambdaisland/clojure-style-guide#whitespace

Reply:
Thanks for the reminder!

Reply:
I fixed this. Let me know if you want me to squash some commits, since a commit just fixing whitespace is pretty trivial.